src/main/java/com/minecolonies/colony/workorders/WorkOrderBuild.java

```diff
@@ -7,6 +7,7 @@
 import com.minecolonies.util.BlockPosUtil;
 import com.minecolonies.util.EntityUtils;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.util.EntityUtils'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)